test/parallel/test-module-relative-lookup.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+require('../common');
+var assert = require('assert');
```

Comment:
As a side effect of using `const`, I had to add a prefix to avoid name collision with `global.module`. Settled for `_module` instead, if you'd rather see another name for it, don't hesitate to say so.